app/controllers/application_controller.rb

```diff
@@ -344,7 +353,7 @@ def analytics_exception_info(exception)
   end
 
   def add_sp_cost(token)
-    Db::SpCost::AddSpCost.call(sp_session[:issuer].to_s, sp_session[:ial2] ? 2 : 1, token)
+    Db::SpCost::AddSpCost.call(sp_session[:issuer].to_s, sp_session_ial_1_or_2, token)
```

Comment:
Ditto, between issuer + raw IAL value, we can construct an `IalContext` object to help clean this up

Reply:
It seems we missed some refactoring and tests when we implemented IAL2 strict. I can create a separate story for this. For now I think we should merge this so we can respond to partners begin start billing for IAL2 step up proofing and IAL2 strict.

Reply:
Ok please lmk what ticket gets filed